LeaderboardBackend/Controllers/AccountController.cs

```diff
@@ -200,12 +201,17 @@ public async Task<ActionResult<LoginResponse>> Login([FromBody] LoginRequest req
     [FeatureGate(Features.ACCOUNT_RECOVERY)]
     public async Task<ActionResult> RecoverAccount(
         [FromServices] IAccountRecoveryService recoveryService,
+        [FromServices] ILogger<AccountController> logger,
         [FromBody] RecoverAccountRequest request
     )
     {
         User? user = await _userService.GetUserByNameAndEmail(request.Username, request.Email);
 
-        if (user is not null)
+        if (user is null)
+        {
+            logger.LogError("User not found: {usermame}", request.Username);
```

Comment:
Also, `usermame`

Reply:
You can't log the ID in that line, since there's no user in the first place. Unless you mean in the context of logging the successful case.

Also yeah I could switch to a warning instead of an error

Reply:
You're right. I was mistaken on that one. Warning or info is more appropriate.